test/perf/dotnet-tools/BenchmarkDotNet.Extensions/CommandLineOptions.cs

```diff
@@ -0,0 +1,97 @@
+using System;
```

Comment:
Missing license?

Reply:
No license is added to files under `dotnet-tools`. The files there are copied from donet/performance, and we may need to sync them from time to time. So, let's just keep them as they are.